src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala

```diff
@@ -224,8 +222,9 @@ sealed trait EmbeddedKafkaSupport {
 
   def startKafka(config: EmbeddedKafkaConfig, kafkaLogDir: Directory = Directory.makeTemp("kafka")): KafkaServer = {
     val zkAddress = s"localhost:${config.zooKeeperPort}"
-
+    
     val properties: Properties = new Properties
+    config.customBrokerProperties.foreach(x => properties.setProperty(x._1,x._2))
```

Comment:
`.foreach { case (key, value) => properties.setProperty(key, value) }` is more expressive